Scripts/Data/Type/CameraList.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace VRTK.Core.Data.Type
+{
+    using UnityEngine;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Provides a collection of scene cameras.
+    /// </summary>
+    public class CameraList : MonoBehaviour

```

Comment:
Change to
```c#
[Serializable]
public class CameraList
```
instead 😄

Reply:
Turns out that's only useful for things you reference on other `MonoBehaviour`s. You can't add `[Serializable]` things as components since they obviously aren't one!